packages/bsky/src/views/index.ts

```diff
@@ -633,6 +633,13 @@ export class Views {
         root = parent
       }
     }
+    if (
+      !state.ctx?.include3pBlocks &&
+      state.postBlocks?.get(parent.uri)?.reply &&
```

Comment:
I think we need to alter `hydratePostBlocks()` to ensure that this the data behind this block is getting pulled.

Reply:
I believe it is! Within `hydratePosts`